src/Components/Title/Recap.js

```diff
@@ -1,69 +1,84 @@
 import React from "react";
-import Anime from 'react-anime';
+import Anime, {anime} from 'react-anime';
 import '../main.css';
 
 class Recap extends React.Component {
+
+    animate = () => {
+        return (this.props.startAnimation) ? {opacity: [0,1], delay: anime.stagger(10000) } : { opacity: [0,0]}
+    }
+
+
+
+    congratsEmoji = () => {
+        return(
+            <div class="column congrats_emoji">
+            <Anime easing="linear" duration="900"
+                loop={true}
+                rotate='-30deg'
+                direction='alternate'
+            >
+                <span role="img" aria-label="congrats" className="congrats_emoji">🎉</span> 
+            </Anime>
+            </div>
+        )
+    }
+
+    // TODO: Make fit into one screen length, where new text replaces the old text
+
     render() {
         return (
         <div>
             <div className="container">
                 <div className="column">
+                    <Anime {...this.animate()}>
```

Comment:
If `this.animate()` is only called once, it might be worth removing the member function and replacing it with an arrow one.